lib/rmt/cli/base.rb

```diff
@@ -90,8 +90,11 @@ def subcommand_prefix
 
   private
 
-  def needs_path(path)
+  def needs_path(path, writable: false)
     raise RMT::CLI::Error.new("#{path} is not a directory.") unless File.directory?(path)
+    if writable
+      raise RMT::CLI::Error.new("#{path} is not accessible. Check permissions.") unless File.writable?(path)
```

Comment:
I suggest to change that line to:
```ruby
      raise RMT::CLI::Error.new("#{path} is not writable by user #{RMT::DEFAULT_USER}.") unless File.writable?(path)
```

This is shorter but more helpful. Since it tells the customer which system user needs to be able to write to that location. The "Check permissions" I removed, since I find it pretty obvious and the format of the output is then more aligned with the style from the output two lines above.

Reply:
I agree mostly, but from my understanding, the `rmt-cli export data <path>` command is run as the user calling the command and not the rmt user? So the error message would be misleading, since the permissions of the rmt user shouldn't matter in this case, but rather the permissions of `$USER`. Or did I get this wrong?